apps/api-extractor/src/api/IExtractorConfig.ts

```diff
@@ -267,6 +267,95 @@ export interface IExtractorTsdocMetadataConfig {
   tsdocMetadataPath?: string;
 }
 
+/**
+ * Used with {@link IExtractorMessageRoutingConfig.logLevel}.
+ *
+ * @public
+ */
+export const enum ExtractorMessageLogLevel {
```

Comment:
Should we include this in node-core-library and tie it to the `Terminal` API?

Reply:
I don't think so. This particular enum describes a file format for API Extractor. I wouldn't want to mix that up with an unrelated package.